test/common/utils/wait_for.js

```diff
@@ -76,8 +80,8 @@ function blockchainReady(cb, retries, timeout, baseUrl) {
 	})();
 }
 
-function nodeStatus(cb) {
-	var request = popsicle.get(`${__testContext.baseUrl}/api/node/status`);
+function nodeStatus(cb, baseUrl) {
```

Comment:
Please make this change in every where introduced a parameter after the callback.

Reply:
Agreed, as a separate PR/issue. Also we should refactor the `blockchainReady` function which has the same problem.